src/main/resources/application-dev.properties

```diff
@@ -1,7 +1,7 @@
 # Database
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 #TODO set port and name for service database
-spring.datasource.url=jdbc:postgresql://localhost:1032/templatedatabase
+spring.datasource.url=jdbc:postgresql://localhost:1032/flashcarddatabase
```

Comment:
Inconsistent name with docker file

Reply:
"flashcard-database" I did that instead of that

Reply:
flashcard_database